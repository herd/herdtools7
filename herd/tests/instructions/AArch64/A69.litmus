AArch64 A69
(* Tests adds (immediate), left shift 12 *)

{ uint64_t 0:X0=42; }

P0;
ADDS X0, X0, #1, lsl #12;

exists (0:X0=4138)
