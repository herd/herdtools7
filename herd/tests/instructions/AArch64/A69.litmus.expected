Test A69 Allowed
States 1
0:X0=4138;
Ok
Witnesses
Positive: 1 Negative: 0
Condition exists (0:X0=4138)
Observation A69 Always 1 0
Hash=2f51de3dcd647233f77732b51012a4ec

